index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <div id="main-header">
+        <div id="logo">
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/5/53/Wikimedia-logo.png"
+            alt="the logo is not available"
+            id="header-img"
+            height="40"
+            width="40"
```

Comment:
# Inline style
I suggest you do not use inline style, it makes your code missy and unreadable.